<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="NICOs-MacBook-Pro.local" name="TestRunner" tests="36" failures="13" timestamp="2022-10-23T12:16:45 SGT" time="1.905" errors="3">
  <testcase name="runScenario" time="0.049" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 400 expected [400] but found [200],
	Error Message is wrong. expected [Invalid name] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 400 expected [400] but found [200],
Error Message is wrong. expected [Invalid name] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidNameErrorMessage(Testcases.java:188)
at ✽.Check hero creation http response with Invalid name error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase06.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.032" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error Message is wrong. expected [Invalid name,Name cannot be blank] but found [Name cannot be blank,Invalid name]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error Message is wrong. expected [Invalid name,Name cannot be blank] but found [Name cannot be blank,Invalid name]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithBlankNameErrorMessage(Testcases.java:170)
at ✽.Check hero creation http response with "Invalid name,Name cannot be blank"(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase05.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.031" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.046" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.029" classname="TestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidFormatSalaryErrorMessage(Testcases.java:312)
at ✽.Check hero creation http response with invalid format salary error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase15.feature:6)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.035" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 400 expected [400] but found [200],
	Error Message is wrong. expected [Invalid salary precision] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 400 expected [400] but found [200],
Error Message is wrong. expected [Invalid salary precision] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidSalaryErrorMessage(Testcases.java:368)
at ✽.Check hero creation http response with invalid salary error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase18.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.054" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.043" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.037" classname="TestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at stepdefs.Testcases.checkHeroCreationHttpResponseWithWrongFormatBrowniePointsMessage(Testcases.java:463)
at ✽.Check hero creation http response with wrong format browniePoints message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase24.feature:6)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.028" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.030" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.035" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.046" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error Message is wrong. expected [Invalid death date format] but found [There are some issues with working class hero]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error Message is wrong. expected [Invalid death date format] but found [There are some issues with working class hero]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidDeathDateErrorMessage(Testcases.java:282)
at ✽.Check hero creation http response with invalid deathDate error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase14.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.414" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.062" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 400 expected [400] but found [200],
	Error Message is wrong. expected [Invalid natid] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 400 expected [400] but found [200],
Error Message is wrong. expected [Invalid natid] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithErrorNatidMessage(Testcases.java:119)
at ✽.Check hero creation http response with error natid message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase03.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.031" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.034" classname="TestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidFormatTaxPaidErrorMessage(Testcases.java:384)
at ✽.Check hero creation http response with invalid format taxPaid error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase19.feature:6)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.051" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 400 expected [400] but found [200],
	Error Message is wrong. expected [Invalid taxPaid precision] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 400 expected [400] but found [200],
Error Message is wrong. expected [Invalid taxPaid precision] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidTaxPaidErrorMessage(Testcases.java:440)
at ✽.Check hero creation http response with invalid taxPaid error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase22.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.055" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.042" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.047" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error Message is wrong. expected [Invalid name,Name cannot be blank] but found [Name cannot be blank,Invalid name]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error Message is wrong. expected [Invalid name,Name cannot be blank] but found [Name cannot be blank,Invalid name]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithBlankNameErrorMessage(Testcases.java:170)
at ✽.Check hero creation http response with "Invalid name,Name cannot be blank"(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase05.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.032" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.028" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.038" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error Message is wrong. expected [Invalid birthdate format] but found [There are some issues with working class hero]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error Message is wrong. expected [Invalid birthdate format] but found [There are some issues with working class hero]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithInvalidBirthdateErrorMessage(Testcases.java:257)
at ✽.Check hero creation http response with invalid birthdate error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase11.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.041" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.036" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.035" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.053" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.053" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error Message is wrong. expected [Invalid gender,Gender cannot be blank] but found [Gender cannot be blank,Invalid gender]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error Message is wrong. expected [Invalid gender,Gender cannot be blank] but found [Gender cannot be blank,Invalid gender]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithBlankGenderErrorMessage2(Testcases.java:232)
at ✽.Check hero creation http response with blank gender error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase08.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.032" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 200 expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 200 expected [200] but found [400]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkTheCorrectHeroDataHttpResponse(Testcases.java:49)
at ✽.Check the correct hero data http response(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase13.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.043" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 400 expected [400] but found [200],
	Error Message is wrong. expected [TaxPaid cannot be blank] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 400 expected [400] but found [200],
Error Message is wrong. expected [TaxPaid cannot be blank] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithBlankTaxPaidErrorMessage(Testcases.java:404)
at ✽.Check hero creation http response with blank taxPaid error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase20.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.062" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 400 expected [400] but found [200],
	Error Message is wrong. expected [TaxPaid cannot be blank] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 400 expected [400] but found [200],
Error Message is wrong. expected [TaxPaid cannot be blank] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkHeroCreationHttpResponseWithBlankTaxPaidErrorMessage(Testcases.java:404)
at ✽.Check hero creation http response with blank taxPaid error message(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase20.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.029" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.047" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	response http code is not 200 expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
response http code is not 200 expected [200] but found [400]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at stepdefs.Testcases.checkTheCorrectHeroDataHttpResponse(Testcases.java:49)
at ✽.Check the correct hero data http response(file:///Users/nico/Desktop/github/oppenheimer-api-test/src/test/resources/features/testcase09.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.087" classname="TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.058" classname="TestRunner"/>
  <system-out/>
</testsuite> <!-- TestRunner -->
